parsons/databases/postgres/postgres.py

```diff
@@ -0,0 +1,74 @@
+from parsons.utilities import check_env
+from parsons.databases.postgres.postgres_core import PostgresCore
+import logging
+import os
+
+
+logger = logging.getLogger(__name__)
+
+
+class Postgres(PostgresCore):
+    """
+    A Postgres class to connect to database. Credentials can be passed from a ``.pgpass`` file
+    stored in your home directory, environmental variables or passed arguments.
+
+    Args:
+        username: str
+            Required if env variable ``PGUSERNAME`` not populated
```

Comment:
This should be `PGUSER` instead of `PGUSERNAME` per the PostgreSQL documentation

> [PGUSER behaves the same as the user connection parameter.](https://www.postgresql.org/docs/12/libpq-envars.html)